README.md

```diff
@@ -27,34 +26,48 @@ Transport compatible with [Apollo](https://github.com/apollographql/subscription
 For just the HTTP middleware:
 >`dotnet add package GraphQL.Server.Transports.AspNetCore`
 
+The HTTP middleware needs an `IGraphQLRequestDeserializer` implementation:
```

Comment:
@benmccallum I would also ask you to note here about this https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to to point reader what to do.

Reply:
How would you like this to look?

Reply:
After line `> (or your own)`:

For more information on how to migrate  from `Newtonsoft.Json` to `System.Text.Json` see [this article](https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to ).

Reply:
Done.